fcrepo4/fcrepo4
# Removed AclRdfContext class.

**JIRA Ticket**: https://jira.duraspace.org/browse/FCREPO-2938

# What does this Pull Request do?

Removed AclRdfContext class that was providing the `fedora:writable` triple.

# What's new?

Since it was doing nothing other than adding the `fedora:writable` triple, removed it entirely. The `ViewHelpers::isWritable` method now always returns true as a stopgap until we have a more robust method of determining the access modes a user is allowed on a resource.

# How should this be tested?

Normal `mvn clean install`. HTML UI sidebar should still be visible when doing `mvn jetty:run -pl fcrepo-webapp`

# Additional Notes:
* Does this change require documentation to be updated? Maybe, if there are references to `fedora:writable` anywhere
* Does this change add any new dependencies? no
* Does this change require any other modifications to be made to the repository (ie. Regeneration activity, etc.)? no
* Could this change impact execution of existing code? only if there is third-party code relying on the presence of a `fedora:writable` triple.

# Interested parties
@dbernstein, @bbpennel, @bseeger, @fcrepo4/committers
